### DISCUSSION TITLE
How do you use the text2rednotebook.py script?

### DISCUSSION THREAD

**Discussion by allstar87** [jendrikseipp/rednotebook#638] [Q&A]
I've got a couple years worth of personal journal in plain text.  I see there's a script here in this repo for converting such things into rednotebook format, but...I don't understand how to use it.

How do you use it?

**Comment by jendrikseipp** [comment]
The script is meant a a starting point for your own script, not an off-the-shelve solution. You'll have to adapt it to your specific input format.

**Reply by allstar87** [reply]
Granted, it says so in the comments.  I imagine it would have to be modified if you've got one file for a year, to split those into 12, or one file per month, or individual day files, how the dates are stored, etc.

What I mean is, I don't get how...Like, are you supposed to put it in the same directory as your journal files and evoke it from there with the file to translate as an argument?

Also, it seems to try to call RedNotebook as if it's a library or something?  How does that work if you've installed from Flatpak?

**Reply by jendrikseipp** [reply]
The script assumes that you cloned the whole repo. Then it'll find RedNotebook as a library automatically.

You can then adapt the script to find your journal files.
